### ISSUE TITLE
RUN.PY FAIL.

### ISSUE THREAD

**Issue by blueveryday** [testerSunshine/12306#456]
**描述问题**
```Could not load dynamic library 'cudart64_100.dll'
A clear and concise description of what the bug is.
```
**重现步骤**
```
步骤一
```
**截图&日志**
D:\GitHub\12306>python run.py
Using TensorFlow backend.
2019-10-10 21:29:29.206135: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudart64_100.dll'; dlerror: cudart64_100.dll not found
2019-10-10 21:29:29.220274: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
WARNING:tensorflow:From D:\GitHub\12306\verify\localVerifyCode.py:14: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From C:\Python37\lib\site-packages\keras\backend\tensorflow_backend.py:3976: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

2019-10-10 21:29:35.467404: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2019-10-10 21:29:35.483233: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'nvcuda.dll'; dlerror: nvcuda.dll not found
2019-10-10 21:29:35.492444: E tensorflow/stream_executor/cuda/cuda_driver.cc:318] failed call to cuInit: UNKNOWN ERROR (303)
2019-10-10 21:29:35.526808: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: hh
2019-10-10 21:29:35.539825: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: hh
WARNING:tensorflow:From C:\Python37\lib\site-packages\keras\backend\tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
C:\Python37\lib\site-packages\keras\engine\saving.py:292: UserWarning: No training configuration found in save file: the model was *not* compiled. Compile it manually.
  warnings.warn('No training configuration found in save file: '
**************************************************
检查当前版本为: 1.1.114
检查当前python版本为：3.7.4，目前版本只支持3.6以上
**环境信息**
 - windows 10
 - python3.7.4
 - 订票小助手版本 1.1.114

**额外的备注**
- Add any other context about the problem here.

**Comment by testerSunshine** [comment]
tf 配置问题，请自行解决
